classes/Warpcast.ts

```diff
@@ -0,0 +1,39 @@
+import { z } from "zod";
+
+// Neynar's api
+const apiUrl = "https://api.neynar.com";
+const apiKey = process.env.NEYNAR_API_KEY as string;
```

Comment:
who's key is this? if we're running this in prod we need a proper account - can you ask Mike to set one up for us?